### PR TITLE
Add Aztec

### PR DESCRIPTION
**Pull request by TheNeverMan**
Add Aztec theme to repository. (https://github.com/TheNeverMan/aztec-base16/)

### CODE REVIEW

File: base16/aztec.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Aztec scheme for the Base16 Builder (https://github.com/chriskempson/base16-builder)
```
Comment (JamyGolden) [inline base16/aztec.yaml]: Can you remove this line?

**Review by JamyGolden — CHANGES_REQUESTED** [review]
Hello :smile: Can you remove the one comment? It doesn't add anything since the information is already contained within the yaml content itself. Once you remove it I'll merge the PR.

File: base16/aztec.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+palette:
+  base00: "101600" # Background ----
+  base01: "1A1E01" # Background ---
+  base02: "242604" # Background --
+  base03: "2E2E05" # Background -
+  base04: "FFD129" # Foreground -
+  base05: "FFDA51" # Foreground --
+  base06: "FFE178" # Foreground ---
+  base07: "FFEBA0" # Foreground ----
+  base08: "EE2E00" # Red
```
Comment (FredHappyface) [inline base16/aztec.yaml:14]: Tbh love the comments here. Cheers :)

File: base16/aztec.yaml (hunk 1 above)
Comment (TheNeverMan) [reply base16/aztec.yaml]: Okay, I just copied yaml template from another theme within the repo, thought it is descriptive and appropriate.

File: base16/aztec.yaml (hunk 2 above)
Comment (TheNeverMan) [reply base16/aztec.yaml:14]: I copied the template from yaml file of 3024 theme. Thank contributor of that theme instead of me.

**Comment by TheNeverMan** [comment]
File is updated. I removed the comment.

**Review by FredHappyface — APPROVED** [review]
lgtm

**Review by JamyGolden — APPROVED** [review]
Nice

**Comment by TheNeverMan** [comment]
It is going to be automatically added to base16 gallery, right?

**Comment by JamyGolden** [comment]
Yeah the builds happen every sunday, but I'll manually run it in the gallery now (the rest of the templates will be updated this sunday)

**Pull request merged**